### ISSUE TITLE
Risk-state unknown after 14 days and switching temporarily to the Swiss Corona App and back

### ISSUE THREAD

**Issue by elektro-wolle** [corona-warn-app/cwa-app-ios#819]
## Avoid duplicates

* [x] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [x] Bug is not already reported in another issue

## Describe the bug

After switching to another contact-tracing app (SwissCovid) after 14 days and returning to the Corona Warn App, the risk-state switches to unknown. 

## Expected behaviour

The previous history of 14 days without contact should not be discarded.

## Steps to reproduce the issue

1. Install CWA as contact tracing app
2. Wait 1 day for retrieving an update
3. Install SwissCovid, enable it as tracing app
4. Immediately switching back to CWA
5. Contact-History lost in CWA  

## Technical details

- iOS Version: 13.5.1
- Device: iPhone 8

## Additional context

The day counter is also reset to zero

![IMG_C45843C2E18E-1](https://user-images.githubusercontent.com/4459943/86237551-41b69b00-bb9c-11ea-9aad-ca5a4dd9f283.jpeg)

But the exposure checks were present:
```
{
  "Build" : "17F80",
  "ExportVersion" : 1,
  "ExposureChecks" : [
    {
      "Hash" : "zzz",
      "RandomIDCount" : 503,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-24 02:20:10 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 503,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-25 02:33:41 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 720,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-25 02:33:41 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 503,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-26 08:12:37 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 720,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-26 08:12:37 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 1160,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-26 08:12:37 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 503,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-27 12:01:55 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 720,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-27 12:01:55 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 1160,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-27 12:01:55 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 1450,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-27 12:01:55 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 503,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-28 16:31:37 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 720,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-28 16:31:37 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 1160,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-28 16:31:37 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 1450,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-28 16:31:37 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 1280,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-28 16:31:37 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 503,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-29 17:05:43 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 720,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-29 17:05:43 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 1160,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-29 17:05:43 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 1450,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-29 17:05:43 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 1280,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-29 17:05:43 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 760,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-29 17:05:43 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 503,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-30 17:48:43 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 720,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-30 17:48:43 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 1160,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-30 17:48:43 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 1450,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-30 17:48:43 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 1280,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-30 17:48:43 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 760,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-30 17:48:43 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 1270,
      "MatchCount" : 0,
      "DataSource" : "de.rki.coronawarnapp",
      "Timestamp" : "2020-06-30 17:48:43 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 10,
      "MatchCount" : 0,
      "DataSource" : "ch.admin.bag.dp3t",
      "Timestamp" : "2020-06-30 18:53:28 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 10,
      "MatchCount" : 0,
      "DataSource" : "ch.admin.bag.dp3t",
      "Timestamp" : "2020-06-30 18:53:28 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 10,
      "MatchCount" : 0,
      "DataSource" : "ch.admin.bag.dp3t",
      "Timestamp" : "2020-06-30 18:53:28 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 10,
      "MatchCount" : 0,
      "DataSource" : "ch.admin.bag.dp3t",
      "Timestamp" : "2020-06-30 18:53:29 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 14,
      "MatchCount" : 0,
      "DataSource" : "ch.admin.bag.dp3t",
      "Timestamp" : "2020-06-30 18:53:29 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 16,
      "MatchCount" : 0,
      "DataSource" : "ch.admin.bag.dp3t",
      "Timestamp" : "2020-06-30 18:53:29 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 14,
      "MatchCount" : 0,
      "DataSource" : "ch.admin.bag.dp3t",
      "Timestamp" : "2020-06-30 18:53:29 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 14,
      "MatchCount" : 0,
      "DataSource" : "ch.admin.bag.dp3t",
      "Timestamp" : "2020-06-30 18:53:29 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 10,
      "MatchCount" : 0,
      "DataSource" : "ch.admin.bag.dp3t",
      "Timestamp" : "2020-06-30 18:53:29 +0200"
    },
    {
      "Hash" : "zzz",
      "RandomIDCount" : 13,
      "MatchCount" : 0,
      "DataSource" : "ch.admin.bag.dp3t",
      "Timestamp" : "2020-06-30 18:53:29 +0200"
    }
  ],
  "DeviceProductType" : "iPhone10,4"
}
```

**Comment by jwildeboer** [comment]
See #805 and install the update 1.0.3. That should fix it.

**Comment by elektro-wolle** [comment]
1.0.3 is currently installed. I'll try waiting until the next exposure check update is delivered.

**Comment by jwildeboer** [comment]
I had to wait for around two hours after the update was installed for the risk calculation to switch from grey to green. FYI.

**Comment by elektro-wolle** [comment]
After waiting a couple of hours, the state switches back to green, but the history is lost:

![IMG_563C7E123F2E-1](https://user-images.githubusercontent.com/4459943/86289700-6e929e80-bbec-11ea-93b9-ddfb080c2c1d.jpeg)
 
Despite the different root cause, this could be closed as duplicate to #805

**Comment by tkowark** [comment]
Thanks again for your patience. As mentioned in other issues, the "history" is only lost as far as the displayed active days are concerned. The RPIs stored in the exposure log of the Exposure Notification Framework are not affected by that.
